src/community/geogig/src/main/java/org/geogig/geoserver/rest/GeoServerRepositoryProvider.java

```diff
@@ -194,5 +221,36 @@ private Repository findRepository(String repositoryName) {
             }
             return Optional.absent();
         }
+
+        static Optional<Repository> importGeogig(Request request) {
```

Comment:
This can be private, it's in a private inner class, not getting exposed outside this class.